packages/l/lv2/package.yml

```diff
@@ -1,15 +1,15 @@
 name       : lv2
 version    : 1.18.10
-release    : 5
+release    : 6
 source     :
     - git|https://github.com/lv2/lv2.git : v1.18.10
+homepage   : https://github.com/lv2/lv2
 license    : ISC
 component  : multimedia.library
 summary    : Open standard for audio plugins
 description: |
     LV2 is an open standard for audio plugins, used by hundreds of plugins and other project. At its core, LV2 is a simple stable interface, accompanied by extensions which add functionality to support the needs of increasingly powerful audio software.
 builddeps  :
-    - pkgconfig(cairo-gl)
```

Comment:
Why this was removed? I feel like you already talk about it, but I don't remember.

Reply:
Yes, this was discussed in Matrix.  

The build failed when looking for cairo-gl as it was removed as part of this [PR](https://github.com/getsolus/packages/commit/9a54bea62349e3bbe5fc820eff0adbadb4a88ba7#diff-5c54e5bd0d1f4607086a73e3a331143fac9f2294f2c7ae842c9f93140c4b96e0L122) per the conversation in Matrix.

EbonJaeger confirmed that it was [not linking against cairgo-gl](https://matrix.to/#/!oxwGTPuyIYEUPjFVbL:matrix.org/$53NhvubJvYqg_OnmKuXLzmijRC8Z6nIQNcGCMrw4z_M?via=matrix.org&via=kde.org&via=envs.net)